boards/arm/atsamr21_xpro/atsamr21_xpro.dts

```diff
@@ -46,6 +54,14 @@
 	clock-frequency = <48000000>;
 };
 
+&tcc0 {
+	status = "okay";
+	compatible = "atmel,sam0-tcc-pwm";
+	/* Gives a maximum period of 1.4s */
```

Comment:
optional: drop the comment.

I added this as a personal note to the Arduino Zero DTS file, but DTS files seem to be spartan and comment free.